.github/workflows/integrationtest.yml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
e.g. https://github.com/pyiron/pyiron/blob/master/.github/workflows/notebooks.yml

Reply:
I am fine with having the integration tests additionally - but then we should also add the notebooks test just like we have it in the other packages.

Reply:
I will add it.